locale/hr_HR/user.po

```diff
@@ -77,8 +77,9 @@ msgstr "Oprostite, poveznica na koju ste kliknuli nije važeća ili nije valjana
 msgid "user.login.lostPassword.invalidUser"
 msgstr "Ne postoji korisnik s navedenom adresom e-pošte."
 
-msgid "user.login.lostPassword.passwordSent"
-msgstr "Nova lozinka je poslana na vašu adresu e-pošte. Možete se sada prijaviti na stranici s vašom novom lozinkom."
+#, fuzzy
+msgid "user.login.resetPassword.passwordUpdated"
+msgstr "Lozinka je uspješno ažurirana. Prijavite se ažuriranom lozinkom."
```

Comment:
Did `user.login.lostPassword.confirmationSentFailedWithReason` and `user.login.passwordResetProcessInstructions` get forgotten here?

Reply:
added. Sorry for the miss .